var/spack/repos/builtin/packages/pipits/package.py

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Pipits(PythonPackage):
+    """Automated pipeline for analyses of fungal ITS from the Illumina
+
+       Note: PIPITS requires that data sets are downloaded manually. The
+       environment variables will be set to the module file automatically by
+       this script, the only thing that will need to be done is placement of
+       these data sets into the appropriate directories. More information about
+       this can be found on PIPITS's homepage."""
```

Comment:
Maybe we can handle this with `resources`? See http://spack.readthedocs.io/en/latest/packaging_guide.html?highlight=resource#resources-expanding-extra-tarballs